Common/CustomException/ForbiddenException.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace CustomException
+{
+    /// <summary>
+    /// Indicates that the server understood the request, but refuses to authorize it
```

Comment:
Not refuses to authorize
I think, there should be something like that: "user must be authorized"

Reply:
Ok, may be my english isn`t so well))

Reply:
May be you should use not authorize or execute